src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -237,6 +238,7 @@ public void ConfigureServices(IServiceCollection services)
                     options.EnableEndpointRouting = false;
                     options.AllowEmptyInputInBodyModelBinding = true;
                 })
+                .AddControllersAsServices()
```

Comment:
Not sure that's true. .NET doesn't have a way to mark a class as a service, so it'd be pretty silly for it to mandate that any class that isn't registered with the container would fail startup (e.g. what about model-like classes).

Anyway, I originally added this because I needed to manually wire specific dependencies into the `BauCacheController`. However, I've just checked and it doesn't seem like adding this actually makes any difference to how the DI works. Have removed it now!